UC NetCode Fall 2024/Assets/App/Resource/Scripts/GameScript.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.Netcode;
+using TMPro;
+using UnityEngine;
+
+namespace App.Resource.Scripts
+{
+    public class GameScript : NetworkBehaviour
+    {
+        [SerializeField] private List<ulong> _currentPlayers;
+
+        public TMP_Text myWinText;
+
+        public override void OnNetworkSpawn()
+        {
+            base.OnNetworkSpawn();
+            myWinText.gameObject.SetActive(false);
+        }
+
+        [Rpc(target:SendTo.Server)]
+
+        public void AddPlayerRpc(RpcParams rpcParams = default)
```

Comment:
The reason why it's not "working" is because we are not calling this in the HealthScript